libs/core/src/lib/info-label/info-label.component.html

```diff
@@ -0,0 +1 @@
+<span class="fd-info-label" [ngClass]=" getCustomIconClass">{{message}}</span>
```

Comment:
Please use the approach the team is using for applying the classes. You can take a look at components like Button, Bar, Link, Form-group, Tab, etc. You need to use the `buildComponentCssClass` method instead of calling `getCustomIconClass`. You can use `<ng-content></ng-content>` instead of `{{message}}`